Formula/dosbox-staging.rb

```diff
@@ -25,15 +27,19 @@ def install
     args = %W[
       --prefix=#{prefix}
       --disable-dependency-tracking
-      --disable-fluidsynth
       --disable-sdltest
       --enable-core-inline
     ]
 
+    ENV.append_to_cflags "-DNDEBUG"
+    ENV.append_to_cflags "-O3"
+    ENV.append_to_cflags "-fno-math-errno"
+    ENV.append_to_cflags "-fno-strict-aliasing"
     system "./autogen.sh"
     system "./configure", *args
-    system "make", "install"
+    system "make", "CXXFLAGS=#{ENV.cflags}", "install"
```

Comment:
Let's try to stick to what upstream recommends in https://github.com/dosbox-staging/dosbox-staging/blob/v0.76.0/README.md#linux-macos:

```
./configure CPPFLAGS="-DNDEBUG" \
            CFLAGS="-O3 -march=native" \
            CXXFLAGS="-O3 -march=native"
```

Reply:
That doesn't work on Apple Silicon. The flags I put in are the flags used if you follow the `scripts/build.sh` script with a "release" target.